runners/core-java/src/main/java/org/apache/beam/runners/core/InMemoryStateInternals.java

```diff
@@ -53,7 +53,7 @@
  * and for running tests that need state.
  */
 @Experimental(Kind.STATE)
-public class InMemoryStateInternals<K> implements StateInternals<K> {
+public class InMemoryStateInternals<K> implements StateInternals {
```

Comment:
Noting that I am not removing all type variables from implementing `StateInternals` as this commit is already large enough, and they actually do expose the key with this type (it is not needed and should likely be removed, but doesn't have to happen now).